CHANGELOG.md

```diff
@@ -8,6 +8,20 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [2.2.1] - 2020-02-04
```

Comment:
Needs a corresponding link at the bottom of the file.

Reply:
:+1: Done